coordinator/testing/pom.xml

```diff
@@ -365,6 +365,7 @@
                     <ccm.version>4.0.10</ccm.version>
                     <stargate.test.nodes>1</stargate.test.nodes>
                     <stargate.test.persistence.module>persistence-cassandra-4.0</stargate.test.persistence.module>
+                    <stargate.test.backend.vsearch>false</stargate.test.backend.vsearch>
```

Comment:
No clean way to auto-detect via driver or ccm so just add another system property for tests